src/main/java/io/github/jidcoo/opto/lcdb/enhancer/core/executor/LeetcodeExecutorFactory.java

```diff
@@ -34,22 +32,22 @@
 public final class LeetcodeExecutorFactory {
 
     /**
-     * Product a LeetcodeExecutor instance by
-     * {@link LeetcodeJavaDebugEnhancer} instance.
+     * Product a LeetcodeExecutor instance by leetcode instance.
      *
-     * @param enhancer the LeetcodeJavaDebugEnhancer instance.
+     * @param instance         the leetcode instance.
+     * @param leetcodeInvokers the leetcode invokers.
      * @return the LeetcodeExecutor instance.
+     * @since 1.0.1
      */
-    public static LeetcodeExecutor getLeetcodeExecutor(LeetcodeJavaDebugEnhancer enhancer) {
-        // Hate the npe.
-        AssertUtil.nonNull(enhancer, "The enhancer cannot be null.");
-        Method enhancementPoint = enhancer.getEnhancementPoint();
-        Object target = enhancer;
-        // Try to look for the inner class Solution in AT if the enhancementPoint is null.
-        if (Objects.isNull(enhancementPoint)) {
-            target = ReflectUtil.resolveSolutionInstance(enhancer);
-            AssertUtil.nonNull(target, "Cannot resolve the inner class Solution from the AT enhancer instance.");
+    public static LeetcodeExecutor getLeetcodeExecutor(Object instance, LeetcodeInvoker... leetcodeInvokers) {
+        AssertUtil.nonNull(instance, "The instance cannot be null.");
+        LeetcodeExecutor executor = new LeetcodeExecutor(instance, null);
```

Comment:
If the parameter `leetcodeInvokers` is not empty here, it must be ensured that the parameter LeetcodeInvoker in constructor: 
`LeetcodeExecutor(Object, LeetcodeInvoker)`
is `leetcodeInvokers[0]`.